tensorflow/c/experimental/profiler/BUILD

```diff
@@ -0,0 +1,64 @@
+#Description
+# Profiler C API
+
+load("//tensorflow/core/platform:rules_cc.bzl", "cc_library")
+
+# buildifier: disable=same-origin-load
+load("//tensorflow:tensorflow.bzl", "filegroup")
+
+package(
+    licenses = ["notice"],
+)
+
+filegroup(
+    name = "headers",
+    srcs = [
+        "profiler.h",
+    ],
+    visibility = ["//tensorflow:__subpackages__"],
+)
+
+cc_library(
+    name = "profiler_hdrs",
```

Comment:
Can you rename this directory and all the files to be "pluggable_profiler" instead of just "profiler"?

Reply:
done